checks/evaluation/signed_releases_test.go

```diff
@@ -15,96 +15,295 @@
 package evaluation
 
 import (
+	"fmt"
 	"testing"
 
-	"github.com/google/go-cmp/cmp"
-	"github.com/google/go-cmp/cmp/cmpopts"
-
 	"github.com/ossf/scorecard/v4/checker"
-	"github.com/ossf/scorecard/v4/clients"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/releasesAreSigned"
+	"github.com/ossf/scorecard/v4/probes/releasesHaveProvenance"
 	scut "github.com/ossf/scorecard/v4/utests"
 )
 
+const (
+	release0 = 0
+	release1 = 1
+	release2 = 2
+	release3 = 3
+	release4 = 4
+)
+
+const (
+	asset0 = 0
+	asset1 = 1
+	asset2 = 2
+	asset3 = 3
+	asset4 = 4
+	asset5 = 5
+	asset6 = 6
+	asset7 = 7
+	asset8 = 8
+	asset9 = 9
+)
+
+func negativeSignedProbe(totalReleases, release, asset int) finding.Finding {
+	return finding.Finding{
+		Probe:   "releasesAreSigned",
+		Outcome: finding.OutcomeNegative,
+		Values: map[string]int{
+			"totalReleases":                   totalReleases,
```

Comment:
let's remove totalReleases?

Reply:
We need that value in the evaluation to calculate the proportional score. We can extract it from the findings, though. WDYT?

Reply:
I'm in favor of removing. We can add later if someone complains or we find out that it makes policy writing (cue, rego) impossible

Reply:
Removed in https://github.com/ossf/scorecard/pull/3610/commits/d30d28bf4b9c7708a4e73efe99abeb18df504805